fetcher/network.go

```diff
@@ -24,26 +24,39 @@ import (
 	"github.com/coinbase/rosetta-sdk-go/types"
 )
 
+type ErrorRes struct {
```

Comment:
I recommend changing this to `type Error` and changing the field names to the ones below:

```suggestion
type Error struct {
	error           error
	clientError *types.Error
}
```

Reply:
I think these fields should be public (i.e. capitalized).